### DISCUSSION TITLE
参考文献编译错误

### DISCUSSION THREAD

**Discussion by chenhongyang113** [sjtug/SJTUThesis#776] [提问] [answered]
[main.log](https://github.com/sjtug/SJTUThesis/files/8329700/main.log)
你好，昨天还好好的，今天在写绪论的地方就开始错误，感觉是参考文献编译的问题，本地是Windows11+VScode，但是我把文件上传overleaf是可以编译的。log文件很多行，不太清楚怎么找错误在哪里，求帮忙~

**Comment by chenhongyang113** [comment, marked as answer]
> [main.log](https://github.com/sjtug/SJTUThesis/files/8329700/main.log) 你好，昨天还好好的，今天在写绪论的地方就开始错误，感觉是参考文献编译的问题，本地是Windows11+VScode，但是我把文件上传overleaf是可以编译的。log文件很多行，不太清楚怎么找错误在哪里，求帮忙~

虽然不知道具体什么问题，但是我把log文件中提到的“{The purpose of this paper is to optimise the hydroelastic performanc\ETC.”这个有关的参考文献删掉就好了。。。那个bibtex明明是艾斯维尔导出的，不懂为啥会报错

**Reply by qiaoxiaoyue** [reply]
我也是同样的问题，原本都编译成功的，突然就参考文献不能编译了。原本清理临时文件，重启电脑是可以的，现在这个方法也不行了。
求大神帮忙。MacOS+VScode

**Reply by chenhongyang113** [reply]
看看log文件倒数后面几行那里有啥错误提示，我的那个问题就是bib文件里最后加的一条参考文献信息有问题。你看我这个log文件最后那里是这样的：

Package biblatex Info: Trying to load bibliographic data...
Package biblatex Info: ... file 'main.bbl' found.
 (./main.bbl)
Runaway argument?
{The purpose of this paper is to optimise the hydroelastic performanc\ETC.
./main.tex:16: File ended while scanning use of \field.
<inserted text> 
                \par 
l.16 \begin{document}
                      
Here is how much of TeX's memory you used:
 47072 strings out of 476919
 953609 string characters out of 5825424
 1809517 words of memory out of 5000000
 66347 multiletter control sequences out of 15000+600000
 407037 words of font info for 60 fonts, out of 8000000 for 9000
 1349 hyphenation exceptions out of 8191
 114i,1n,142p,1991b,844s stack positions out of 5000i,500n,10000p,200000b,80000s

No pages of output.


然后这句“{The purpose of this paper is to optimise the hydroelastic performanc\ETC.”的参考文献我删掉就可以正常编译了，bib源可以试试用谷歌学术的，我那条用的是爱思唯尔的，不知道为啥那条有错误

**Reply by qiaoxiaoyue** [reply]
我的不是加入参考文献的问题，应该是清理了参考文献缓存文件，然后写新的东西时没有引入参考文献文件，找不到对应的参考文献报错的，现在好了。
